script/Ciel/Day 8/08_01_CIEL08_3B.txt

```diff
@@ -237,17 +238,18 @@ The butter chicken was finger-licking good.
 [sha:5ba48769cc2c8e32b2e66e9dfddc959ec5a7908f]{
 -- Page 267, Offset 18934.
 -- カレーソースはバターが効いているのか辛いのに甘いという絶妙なバランスで、ナンも熱く柔らかく、パン生地を噛みしめるだけで旨みが口内に溢れてくる。
-The curry sauce was an exquisite balance of spiciness and sweetness, thanks perhaps to the butter. The naan was pipping hot and soft so that just by taking a bite of the bread, I could feel the savoriness spreading all over my taste buds.
+// 旨み to "savoriness" here, dunno if umami is common knowledge enough to keep as is
+The curry sauce was an exquisite balance of spiciness and sweetness, perhaps thanks to the butter. The naan was piping hot and soft, and simply biting the dough made the savoriness permeate throughout my mouth.
 }
 [sha:17d6e9ac63639efb9d7301eae9cf2e06e1a64638]{
 -- Page 268, Offset 18935.
 -- この店は当たりだ。
-That restaurant is a gem.
+This restaurant is a gem.
 }
 [sha:1fb24a817b52c5ff99c65ab7b3503f2b6febb22e]{
 -- Page 268, Offset 18936.
 -- カレーショップ・メシアン、覚えておこう。有彦にも教えてやろう。
-I'll keep note of you, curry shop 'Messiaen'. I'll recommend it to Arihiko too.
+I'll remember you, curry shop 'Messiaen'. I'll tell Arihiko about it too.
```

Comment:
is this supposed to be curry shop messiah? Or is he referencing the composer? this is a weird line

Reply:
i had to do a double take too, but the name is correct